src/main/resources/db/migration/V2__add_job_contexts.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE job_contexts (
+  uuid       UUID PRIMARY KEY,
+  created_at TIMESTAMP NOT NULL,
+  context    TEXT NOT NULL,
```

Comment:
@julienledem @fanrukun, have any strong options on what type we should use for storing `JSON`? That is, `TEXT` vs `VARCHAR`

Reply:
https://stackoverflow.com/questions/25300821/difference-between-varchar-and-text-in-mysql

I use varchar most of the time (as it can specify the length limit for efficiency).  I guess in your case both looks fine.

Reply:
Given we're unsure of what the minimum size (in _bytes_) for `context` will be (see #637 ), using `TEXT` provides us the flexibility for now. We can enforce the size constraint in the app layer.